R/return_linter.R

```diff
@@ -130,6 +146,12 @@ return_linter <- function(
 
   Linter(linter_level = "expression", function(source_expression) {
     xml <- source_expression$xml_parsed_content
+    if (defer_except) {
+      assigned_functions <- xml_text(xml_find_all(xml, function_name_xpath))
+      except <- union(except, assigned_functions[re_matches(assigned_functions, except_regex)])
+      except_xpath <- glue(except_xpath_fmt)
+      body_xpath <- glue(body_xpath_fmt)
```

Comment:
Using named dots for glue like `glue::glue("{a}{b}", a = 1, b = 2)` would make the implicit dependency of `body_xpath` on `except_xpath` explicit (also, this would remove the object usage false positives). WDYT about that?

Reply:
Great idea, much improved. Hopefully I didn't screw it up in the browser :)